src/mongocxx/bulk_write.cpp

```diff
@@ -33,8 +33,16 @@ bulk_write& bulk_write::operator=(bulk_write&&) noexcept = default;
 
 bulk_write::~bulk_write() = default;
 
-bulk_write::bulk_write(bool ordered)
-    : _impl(stdx::make_unique<impl>(libmongoc::bulk_operation_new(ordered))) {
+bulk_write::bulk_write(options::bulk_write options)
+    : _impl(stdx::make_unique<impl>(libmongoc::bulk_operation_new(options.ordered()))) {
+    auto options_wc = options.write_concern();
```

Comment:
The "ordered" parameter isn't stored or returned as an optional anymore.  I'm wondering if we want to make the decision about default ordered at the options class level, the way it is now, or defer that decision, the way it was before?